service/src/main/java/bio/terra/tanagra/app/controller/ReviewsApiController.java

```diff
@@ -109,8 +109,7 @@ public ResponseEntity<ApiReview> createReview(
                 .size(body.getSize()),
             SpringAuthentication.getCurrentUser().getEmail(),
             entityFilter);
-    return ResponseEntity.ok(
-        toApiObject(createdReview, cohortService.getCohort(studyId, cohortId)));
```

Comment:
I don't think this will work, because the cohort's most recent revision will have updated after creating the review, so you have to re-fetch it.

Reply:
reverted